docs/topics/outlines.rst

```diff
@@ -0,0 +1,75 @@
+.. _outlines:
+
+Outlines
+========
+Outlines (sometimes also called *bookmarks*) are shown in a the PDF viewer
+aside of the page, allowing for navigation within the document.
+
+Creating outlines
+-----------------
+Outlines can be created from scratch, e.g. when assembling a set of PDF files
+into a single document.
+
+.. ipython::
+    :verbatim:
+
+    In [1]: from glob import glob
+
+    In [1]: from pikepdf import Pdf, Outline, OutlineItem
+
+    In [1]: pdf = Pdf.new()
+
+    In [1]: outlines = Outline(pdf)
+
+    In [1]: page_count = 0
+
+    In [1]: for file in glob('*.pdf'):
```

Comment:
This is a useful example and should be kept, but I think the first example should just show how to set a few bookmarks on known pages of interest in a single file, for simplicity. (e.g. we want to bookmark pages 1, 3, and 9).

Reply:
I have added a more straightforward example before this one now.